src/v0/destinations/ga4_v2/customMappingsHandler.js

```diff
@@ -145,7 +145,6 @@ const handleCustomMappings = (message, Config) => {
 const boilerplateOperations = (ga4Payload, message, Config, eventName) => {
   removeReservedParameterPrefixNames(ga4Payload.events[0].params);
   ga4Payload.events[0].name = eventName;
-  const integrationsObj = getIntegrationsObj(message, 'ga4_v2');
```

Comment:
We can remove import of `getIntegrationsObj` as well from util.

Reply:
removed